go.mod

```diff
@@ -18,4 +18,5 @@ require (
 	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
 	sigs.k8s.io/controller-runtime v0.2.0-beta.2
+	sigs.k8s.io/controller-tools v0.2.0-beta.2 // indirect
```

Comment:
do we need this? wonder what change caused this to get added?

Reply:
This is after I removed controller-gen from my previous installation to let make file get the version included.

```
BANLW1AD4EFD4A:instance-manager nmogulla$ make
go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.0-beta.2
go: finding sigs.k8s.io v0.2.0-beta.2
go: finding sigs.k8s.io/controller-tools/cmd/controller-gen v0.2.0-beta.2
go: finding sigs.k8s.io/controller-tools/cmd v0.2.0-beta.2
go: finding sigs.k8s.io/controller-tools v0.2.0-beta.2
go: downloading sigs.k8s.io/controller-tools v0.2.0-beta.2
go: extracting sigs.k8s.io/controller-tools v0.2.0-beta.2
go: downloading github.com/spf13/cobra v0.0.5
go: downloading github.com/markbates/inflect v1.0.4
go: downloading golang.org/x/tools v0.0.0-20190501045030-23463209683d
go: extracting github.com/spf13/cobra v0.0.5
go: extracting github.com/markbates/inflect v1.0.4
go: downloading github.com/gobuffalo/envy v1.6.15
go: downloading github.com/inconshreveable/mousetrap v1.0.0
go: extracting github.com/inconshreveable/mousetrap v1.0.0
go: extracting github.com/gobuffalo/envy v1.6.15
go: downloading github.com/joho/godotenv v1.3.0
go: downloading github.com/rogpeppe/go-internal v1.2.2
go: extracting github.com/joho/godotenv v1.3.0
go: extracting github.com/rogpeppe/go-internal v1.2.2
go: extracting golang.org/x/tools v0.0.0-20190501045030-23463209683d
go: finding golang.org/x/tools v0.0.0-20190501045030-23463209683d
go: finding github.com/markbates/inflect v1.0.4
go: finding github.com/gobuffalo/envy v1.6.15
go: finding github.com/joho/godotenv v1.3.0
go: finding github.com/rogpeppe/go-internal v1.2.2
/Users/nmogulla/Desktop/Eclipse_Workspace/GoProjects2/bin/controller-gen object:headerFile=./hack/boilerplate.go.txt paths=./api/...
go fmt ./...
go vet ./...
go build -o bin/manager main.go

Reply:
Can you try removing from that line `go.mod`? and running `go mod tidy`. does things still work for you?
looks like it has a bunch of dependency additions in `go.sum`

Reply:
Done!